src/models/photos.model.js

```diff
@@ -0,0 +1,37 @@
+const mongoose = require("mongoose");
+
+const photoSchema = new mongoose.Schema({
+  info: {
+    type: [
+      {
+        url: {
+          type: String,
+        },
+        createdBy: {
+          type: String,
+        },
+        likes: {
+          type: [
+            {
+              type: String,

```

Comment:
Que es tipo de likes ?. Hay diferentes tipos de likes?. Aparte como sabes quien dio el like, cuando un usuario entra como sabe que ya le dio like a una foto.
```suggestion
              userId: String,  
```

Reply:
No, sólo es uno jeje. Ya lo modifiqué